src/types.ts

```diff
@@ -6,14 +6,16 @@ type FunctionOutputInputValues = {
   [key: string]: unknown;
 };
 
-export type InvocationPayload<Body> = {
+export type InvocationPayload<Body extends ValidInvocationPayloadBody> = {
   body: Body;
   context: {
     bot_access_token: string;
     variables: EnvironmentVariables;
   };
 };
 
+export type ValidInvocationPayloadBody = FunctionInvocationBody;
```

Comment:
This will get extended to support new interactivity event payloads as well, so just setting it up.